config.yml

```diff
@@ -211,45 +208,52 @@ steps:
         # End filepath verification
         #######################################################
       - type: respond
-        issue: 3
+        issue: "%registration.store.secondIssueNumber%"
         with: new-pull-request.md
         data:
-          pullUrl: "%payload.pull_request.html_url%"
+          pullUrl: "%registration.store.secondPullUrl%"
       - type: closeIssue
-        issue: 3
+        issue: "%registration.store.secondIssueNumber%"
       - type: createReview
         body: 04_merge-workflow.md
         event: APPROVE
+        # store:
+        #   secondPullNumber: "%result.data.number%"
+        #   secondPullUrl: "%result.data.html_url%"
         data:
           actionsUrl: "%payload.repository.html_url%/actions"
 
-  - title: Merge the new workflow into the master branch
+  - title: Merge the updated workflow into the master branch
     description: Make the workflow available for use on the repository
     event: pull_request.closed
-    link: "{{ repoUrl }}/issues/5"
+    link: "{{ store.secondPullUrl }}"
     actions:
       - type: createIssue
-        title: Create an issue comment with GitHub Script
+        title: Comment and triage with GitHub Script
         body: 05_workflow-triggered.md
         action_id: triggerIssue
+        store:
+          thirdIssueUrl: "%result.data.html_url%"
+          thirdIssueNumber: "%result.data.number%"
       - type: respond
         with: new-issue.md
-        issue: Update my-workflow.yml
+        issue: "%registration.store.secondPullNumber%"
         data:
-          issueURL: "%actions.triggerIssue.data.html_url%"
+          issueURL: "%registration.store.thirdIssueUrl%"
 
   - title: Break your workflow into multiple steps
     description: Separate the workflow concerns by adding more steps
-    event: check_suite.completed
-    link: "{{repoUrl}}/issues/5"
+    # event: check_suite.completed
+    event: issue_comment.created
+    link: "{{store.thirdIssueUrl}}"
     actions:
       - type: gate
-        left: "%payload.check_suite.app.name%"
+        left: "%payload.comment.user.login%"
         operator: ===
-        right: "GitHub Actions"
+        right: "github-actions[bot]"
```

Comment:
This is another case where I think it makes a lot more sense for the learner to trigger the response than for the bot. The way I see it, it kind of takes all the fun out of creating my own workflow and scripts if a bot is causing all the fun stuff to happen instead of me!

Reply:
at the end of the course the learner opens their own issue to demonstrate not supplying a tag. again I echo my feelings about issues not being the focal point of the course.

Reply:
I hear ya, but as a learner that took your course for the first time, I didn't really understand what was going on when things were just automatically happening. This wasn't a good learning experience for me. I ultimately leave it up to you, which is what I left these notes as comments and not direct commits, but that's my feedback!

Reply:
I appreciate your feedback!  I'm gonna see if I can get some more opinions on this before deciding one way or the next.